task_1/anna.zinovatnaya/src/main/java/task_1/Dialog.java

```diff
@@ -0,0 +1,272 @@
+package main.java;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Vector;
+
+public class Dialog {
+
+  /*
+  Files and Folders represent real files that are under git. If two files are equal they are still stored two times
+   */
+  private Folder currentFolder;
+  private Folder rootFolder; // modified area
+
+  /*
+  Map stores committed elements (files, folders and commits). If two elements are equal, they are stored only once.
+   */
+  private Map<String, String> commitedFilesMap; // committed area
+
+  /*
+  Vector stores history of commits
+  */
+  private Vector<Commit> commits;
+
+
+
+  public Dialog() {
+    this.currentFolder = new Folder();
+    this.rootFolder = currentFolder;
+    this.commitedFilesMap = new HashMap<>();
+    this.commits = new Vector<>();
+  }
+
+
+
+  void start() {
+
+    String option = "0";
+
+    Scanner scanner = new Scanner(System.in, "UTF-8");
+
+    System.out.println("Commands:");
+    System.out.println("");
+    System.out.println("Type 'new file' to create new file in a current folder");
+    System.out.println("Type 'new folder' to create new folder in a current folder");
+
+    System.out.println("Type 'current folder' to get current folder hash code");
+    System.out.println("Type 'root folder' to get root folder hash code");
+    System.out.println("Type 'change folder' to go to another folder");
+
+    System.out.println("Type 'commit' to commit files");
+
+    System.out.println("Type 'log' to see history of commits");
+
+    System.out.println("Type 'read not committed element' to read certain element");
+    System.out.println("Type 'read committed element' to read certain committed element");
+
+    System.out.println("Type 'read not committed elements' to read all not committed elements");
+    System.out.println("Type 'read committed elements' to read all committed elements");
+
+    System.out.println("Type 'exit' to exit");
+
+
+    boolean endOfDialog = false;
+
+    while (endOfDialog == false) {
```

Comment:
Such statement looks better as **while (!endOfDialog)**. It is not good approach to compare booleans in a way you did.